source/System.cpp

```diff
@@ -303,6 +308,11 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				object.message = &UNINHABITEDPLANET;
 		}
 	}
+	
+    // Make sure that the stellarWindStrength and luminosity are not game brakeingly low or high.
+    stellarWindStrength = std::max(0.25, stellarWindStrength);
+    luminosity = std::max(0.5, luminosity);
+    luminosity = std::min(2.0, luminosity);
```

Comment:
You are correct.. Even with -.05 stellar wind, with 5 wanderer ramscoops at the center of the stem the player would only be losing around .8 fuel per second. The drain can generally be as high as 20 fuel/s before the player can not notice in time (much like how the player can be eaten by nanobots if too much time is spent looking around)